web/cobrands/northamptonshire/layout.scss

```diff
@@ -1,6 +1,24 @@
 @import "_colours";
 
 @import "../sass/layout";
+@import "./docs/theme/sass/_variables";
+
+#site-logo {
+    width: 450px;
+    
+    // Prevents logo overlapping nav items at medium screen widths    
+    @media (max-width: 60em) {
+        width: 260px;
+    }
+}
+
+#site-header {
+    padding: 20px 0;
```

Comment:
Looks like somebody forgot to check what effect this would have on the map page header ;-)

![Screenshot_2021-03-16 Viewing a location](https://user-images.githubusercontent.com/739624/111334345-65a70780-866b-11eb-9902-fd577510e021.png)

You can set a custom `$mappage-header-height` in `web/cobrands/northamptonshire/_colours.scss` to control the height of the header on the map page. It defaults to `60px`, but given the padding you’ve added to `#site-header`, it looks like you need something like `104px`.